sparkler-deployment/docker/Dockerfile

```diff
@@ -66,6 +70,7 @@ COPY ./sparkler-ui/target/sparkler-ui-*.war /data/solr/server/solr-webapp/sparkl
 RUN cp /data/sparkler/conf/solr/sparkler-jetty-context.xml /data/solr/server/contexts/
 
 # Fix permissions
+RUN chmod -R 777 /data/
```

Comment:
@prenastro 
`chmod -R 777` should be avoided. 
The below line `chown -R sparkler:sparkler /data/` gives permission to `sparkler` user.  What was the issue here, are you using a different user?

Reply:
Hi @thammegowda, I tried installing using apt-get install command inside the container and it was throwing me Permission denied error. I think it was because of user issue. Now that is fixed because I am installing it directly from Dockerfile. Should I remove chmod command?